include/psp2/promoterutil.h

```diff
@@ -59,7 +68,7 @@ int scePromoterUtilityUpdateLiveArea(ScePromoterUtilityLAUpdate *args);
  *
  * @return 0 on success.
  */
-int scePromoterUtilityPromoteImport(const char *path);
+int scePromoterUtilityPromoteImport(ScePromoterUtilImportParams *params);
```

Comment:
Could you update the documentation of this function? For example add "it can be used to install games WITHOUT a valid license file" if it's true.